### ISSUE TITLE
enable addons kube-dashboard

### ISSUE THREAD

**Issue by magikcypress** [Azure/azure-cli#11825]
## Describe the bug

**Command Name**
`az aks enable-addons`

**Errors:**
```
argument of type 'NoneType' is not iterable
Traceback (most recent call last):
python3.6/site-packages/knack/cli.py, ln 206, in invoke
    cmd_result = self.invocation.execute(args)
cli/core/commands/__init__.py, ln 608, in execute
    raise ex
cli/core/commands/__init__.py, ln 666, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
cli/core/commands/__init__.py, ln 659, in _run_job
    six.reraise(*sys.exc_info())
...
cli/command_modules/acs/custom.py, ln 1844, in aks_enable_addons
    _ensure_container_insights_for_monitoring(cmd, instance.addon_profiles['omsagent'])
cli/command_modules/acs/custom.py, ln 2487, in _ensure_container_insights_for_monitoring
    if 'loganalyticsworkspaceresourceid' in addon.config:
TypeError: argument of type 'NoneType' is not iterable
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az aks enable-addons --addons {} --resource-group {} --name {}`

## Environment Summary
```
Linux-4.19.0-6-amd64-x86_64-with-debian-10.2
Python 3.6.5
Shell: bash

azure-cli 2.0.79

```

**Comment by tobyhersey** [comment]
Same issue here with , azure-cli  2.0.75 * on macos 
Python (Darwin) 3.7.4

**Comment by edjwhho** [comment]
same error on ubuntu:
azure-cli                         2.0.81
command-modules-nspkg              2.0.3
core                              2.0.81
nspkg                              3.0.4
telemetry                          1.0.4

**Comment by jluk** [comment]
Thank you for the report, this looks like the same issue being reported on the separate issue below. We believe we've found the bug, goal of fix on the next cli release version 2.0.82.

https://github.com/Azure/azure-cli/issues/11456

**Comment by jluk** [comment]
A fix has been made, please reference this issue for the latest tracking. I am going to close this to consolidate info to one thread.

https://github.com/Azure/azure-cli/issues/11456
